runtime/lib/component/custom-config.js

```diff
@@ -0,0 +1,97 @@
+'use strict'
+var logger = require('logger')('custom-config-runtime')
+var config = require('/etc/yoda/custom-config.json')
+var querystring = require('querystring')
+var safeParse = require('@yoda/util').json.safeParse
+
+/**
+ * handle all custom-config
+ */
+class CustomConfig {
+  constructor (runtime) {
+    this.runtime = runtime
+  }
+
+  /**
+   * interce the configuration message for dnd-model etc
+   * @param {string|object} msg
+   */
+  interce (msg) {
```

Comment:
What does 'interce` stand for?

Reply:
dnd-mode was included in the custom-config, but it's a component now, so we should intercept the dnd-mode configuration, then dispatch to the component, not the custom-config app.